cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -42,6 +43,8 @@
     'weber': 1635923188204,  # 2021-11-03 07:06:28.204 UTC
 }
 
+ZPHASE_DATA = {'rainbow': 'rainbow_zphase.json', 'weber': 'weber_zphase.json'}
+
```

Comment:
Do we need to add these files to a package somewhere so they get packaged up correctly?

Reply:
These are under `cirq/devices/calibrations`, which is already packaged in setup.py.